scripts/install-remote-ssh.sh

```
@@ -0,0 +1,54 @@
+# host, where the extension landed there instead and so was invisible to
+# the actual remote extension host.
+set -e
+cd "$(dirname "$0")"
+
+VSIX=$(ls -1 *.vsix 2>/dev/null | head -n 1)
+if [ -z "$VSIX" ]; then
+    echo "No .vsix file found next to this script."
+    exit 1
+fi
+
+REMOTE_CLI=$(ls -t "$HOME"/.vscode-server*/code-* "$HOME"/.vscode-server*/bin/*/bin/remote-cli/code 2>/dev/null | head -n 1)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)